lib/index.ts

```diff
@@ -166,6 +168,14 @@ async function main() {
 
     client.on('ready', async () => {
         console.log('Logged in');
+
+        function runner() {
+            handleFloorPriceChannel(client);
+            handleTotalVolumeChannel(client);
```

Comment:
Not really a big issue but both of these functions request the same data from ME, perhaps we should call `handleGetFromME` first and pass the result to these functions instead of hitting the endpoint twice?